mcp/gcp_ae.tf

```diff
@@ -377,23 +343,18 @@ resource "google_app_engine_standard_app_version" "self" {
   instance_class = lookup(each.value, "instance_class", null)
   runtime_api_version = lookup(each.value, "runtime_api_version", null)
   threadsafe = lookup(each.value, "threadsafe", null)
-  env_variables = merge(
```

Comment:
I had misunderstood this, and thought that it was redundant since the locals section merges the `common`. But realised that this shouldnt have been deleted since it allows for `env_variables` to be declared in both `common` and `spec` and all those settings to be included, not just the ones in `spec`.
I will revert this change

Reply:
okay. Is there not a test that should have picked this up?

Reply:
No there isnt because this merging happens in the actual adapter rather than in locals